jenkins/Jenkinsfile

```diff
@@ -84,5 +84,90 @@ pipeline {
         }
       }
     }
+    stage('Build Images') {
+      parallel {
+        stage('Build bootstrap-tools container') {
+          options {
+            skipDefaultCheckout true
+          }
+          agent {
+            kubernetes {
+              slaveConnectTimeout 240000
+              activeDeadlineSeconds 43200
+              yamlFile 'jenkins/packaging-pod.yaml'
+            }
+          }
+          steps {
+            container('jnlp'){
+              unstash name: 'sources'
+              unstash name: 'ops_data'
+            }
+            container('kaniko'){
+              sh '''
+                /kaniko/executor -f $WORKSPACE/ops/Dockerfile-bootstrap-tools -c $WORKSPACE/ops \
+                  --destination=$DOCKER_OWNER/$BOOTSTRAP_TOOLS_DOCKER_REPO:${NETWORK}-${TAG_VERSION} \
+                  --destination=$DOCKER_OWNER/$BOOTSTRAP_TOOLS_DOCKER_REPO:${NETWORK}-latest
+              '''
+            }
+          }
+        }
+        stage('Build Consensus Service container') {
+          options {
+            skipDefaultCheckout true
+          }
+          agent {
+            kubernetes {
+              slaveConnectTimeout 240000
+              activeDeadlineSeconds 43200
+              yamlFile 'jenkins/packaging-pod.yaml'
+            }
+          }
+          steps {
+            container('jnlp'){
```

Comment:
Whats jnlp?

Reply:
JNLP is the default container for Jenkins running on a build-pod.  Its guaranteed to have basic UNIX tools.  In this case, it has the `tar` command  that is used to `unstash` files from a previous build step.

Reply:
See: https://github.com/jenkinsci/kubernetes-plugin

Reply:
Cool, thanks for explaining.